Error when compiling fil_backend

Triton-Inference-Server is seeing an error when attempting to compile the fil_backend. Below is the error given by cmake:
```
[build-stage 3/4] RUN source /conda/dev/bin/activate  && cmake       --log-level=VERBOSE       -GNinja       -DCMAKE_BUILD_TYPE="Release"       -DBUILD_TESTS=""       -DTRITON_CORE_REPO_TAG="main"       -DTRITON_COMMON_REPO_TAG="main"       -DTRITON_BACKEND_REPO_TAG="main"       -DTRITON_ENABLE_GPU="ON"       -DTRITON_ENABLE_STATS="ON"       -DRAPIDS_DEPENDENCIES_VERSION="24.04"       -DTRITON_FIL_USE_TREELITE_STATIC="ON"       -DCMAKE_INSTALL_PREFIX=/rapids_triton/install       ..  && cd _deps/treelite-src  && git apply /rapids_triton/0001-Allow-predicting-with-FP32-input-and-FP64-models.patch  && cd ../..:

...


#32 43.85 CMake Error at build/_deps/raft-src/cpp/CMakeLists.txt:667 (target_link_libraries):
#32 43.85   The link interface of target "raft_distributed" contains:
#32 43.85 
#32 43.85     ucx::ucp
#32 43.85 
#32 43.85   but the target was not found.  Possible reasons include:
#32 43.85 
#32 43.85     * There is a typo in the target name.
#32 43.85     * A find_package call is missing for an IMPORTED target.
#32 43.85     * An ALIAS target is missing.
#32 43.85 
#32 43.85 
#32 43.85 
#32 43.85 CMake Generate step failed.  Build files cannot be regenerated correctly.
```

This failed our nightly build and looks to be related to the following change: https://github.com/triton-inference-server/fil_backend/pull/394

Let us know of any more information you need to troubleshoot, and thanks in advance for the help!

--- a/conda/environments/rapids_triton_dev.yml
+++ b/conda/environments/rapids_triton_dev.yml
@@ -4,6 +4,6 @@ channels:
   - conda-forge
 dependencies:
   - ccache
-  - cmake>=3.26.4
+  - cmake>=3.26.4,!=3.30.0
   - ninja
   - rapidjson>=1.1.0,<1.1.0.post*

